kolibri/plugins/facility/assets/src/views/users/sidePanels/UserCreate/ClassesSelect.vue

```diff
@@ -55,7 +55,7 @@
         }));
 
         classesOptions.unshift(allClassesOption.value);
-        return classesOptions;
+        return classesOptions.sort((a, b) => a.label.localeCompare(b.label));
```

Comment:
Looks good!, but we should sort these options before we unshift the allClassesOption, as it should be always at the top